packages/backend-modules/auth/graphql/resolvers/User.js

```diff
@@ -83,4 +84,7 @@ module.exports = {
       })
     }
   },
+  alias: async (user, _, { pgdb }) => {
+    return user.alias || (await generateUserAlias(user, pgdb))
```

Comment:
I would only return the user alias for your own user and for admin/supporter roles. Maybe something like
```suggestion
    if (Roles.userIsMeOrInRoles(user, me, DEFAULT_USER_ACCESS_ROLES)) {
        return user.alias || (await generateUserAlias(user, pgdb))
    }
```